WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -356,6 +364,22 @@ def execute( self ):
       self.log.error( "Could not configure DIRAC basics [ERROR %d]" % retCode )
       self.exitWithError( retCode )
 
+  def _genericOptionsCFG(self):
+    """ Append pilot generic options
+    """
+    modList = []
+    for opt in self.pp.genericOptions:
+      key,value = opt.split('=')
```

Comment:
in one line: `for key, value in (opt.split('=') for opt in self.pp.genericOptions):`